sklearn/tests/test_naive_bayes.py

```diff
@@ -409,19 +419,6 @@ def test_discretenb_sample_weight_multiclass(cls):
     assert_array_equal(clf.predict(X), [0, 1, 1, 2])
 
 
-@pytest.mark.parametrize('cls', [BernoulliNB, MultinomialNB])
```

Comment:
We should keep these and add a comment to remove the test when `coef_` is deprecated.

We can add a `@ignore_warnings(category=FutureWarning)` to catch the warnings.

Reply:
Done!